syft/frameworks/crypten/context.py

```diff
@@ -159,6 +160,8 @@ def wrapper(*args, **kwargs):
             if was_initialized:
                 crypten.init()
 
+            sy.local_worker.remove_crypten_support()
```

Comment:
I wonder if we should remove crypten support?

Reply:
So you suggest the following:
If we want to run with a plan/jail we add the support to workers and this support should not be removed, until explicitly the user requests that

Reply:
### TODO: Don't remove this.